go-controller/hybrid-overlay/pkg/controller/node_linux.go

```diff
@@ -656,3 +741,12 @@ func (n *NodeController) updateFlowCacheEntry(cookie string, flows []string, ign
 	n.flowCache[cookie] = &flowCacheEntry{flows: flows}
 	n.flowCache[cookie].ignoreLearn = ignoreLearn
 }
+
+func makeRoute(dstSubnet *net.IPNet, drIP net.IP, mgmtPortIndex int) *netlink.Route {
```

Comment:
@JacobTanenbaum I would just pass the *netlink.Link here and then the function can do the Attrs().Index

Reply:
done